packages/core/src/Stores/client-store.js

```diff
@@ -1774,6 +1776,32 @@ export default class ClientStore extends BaseStore {
         const is_virtual = account.is_virtual;
         const account_type = !is_virtual && currency ? currency : this.account_title;
 
+        const ppc_campaign_cookies =
+            Cookies.getJSON('utm_data') === 'null'
+                ? {
+                      utm_source: 'no source',
+                      utm_medium: 'mo medium',
+                      utm_campaign: 'no campaign',
+                  }
+                : Cookies.getJSON('utm_data');
+        const broker = LocalStore?.get('active_loginid')
+            ?.match(/[a-zA-Z]+/g)
+            ?.join('');
+        setTimeout(() => {
+            Analytics.setAttributes({
+                account_type: broker === 'null' ? 'unlogged' : broker,
+                app_id: String(getAppId()),
+                device_type: isMobile() ? 'mobile' : 'desktop',
+                language: getLanguage(),
+                device_language: navigator?.language || 'en-EN',
+                user_language: getLanguage().toLowerCase(),
+                country: Cookies.get('clients_country') || Cookies?.getJSON('website_status'),
+                utm_source: ppc_campaign_cookies?.utm_source,
+                utm_medium: ppc_campaign_cookies?.utm_medium,
+                utm_campaign: ppc_campaign_cookies?.utm_campaign,
+            });
+        }, 4);
+
```

Comment:
This is fine for now, but I think we need to cover this issue in our `analytics` package. The problem is, everytime we are calling `setAttributes` function the existing attributes gets reset and we have to pass all the attributes again when we call `setAttributes` function (refer to the image below). we must use the following code in our library:

```typescript

    setAttributes({ id, country, user_language, device_language, device_type }: TGrowthbookAttributes) {
        return this.GrowthBook.setAttributes({
            ...this.GrowthBook.getAttributes(),
            id,
            ...(country !== undefined && { country }),
            ...(user_language !== undefined && { user_language }),
            ...(device_language !== undefined && { device_language }),
            ...(device_type !== undefined && { device_type }),
        })
    }
```
<img width="991" alt="image" src="https://github.com/binary-com/deriv-app/assets/129206554/656ed678-e7a8-4608-b2e6-5808b57fc0e1">